packages/devtools-reps/src/reps/stubs/element-node.js

```diff
@@ -206,6 +206,22 @@ stubs.set("SvgNodeInXHTML", {
   }
 });
 
+stubs.set("NodeWithLongAttribute", {
+  type: "object",
+  actor: "server1.conn1.child1/obj32",
+  class: "HTMLParagraphElement",
+  ownPropertyLength: 0,
+  preview: {
+    kind: "DOMNode",
+    nodeType: 1,
+    nodeName: "p",
+    attributes: {
+      "data-test": "a".repeat(100)
```

Comment:
I don't think this qualifies as a `LongString`, as we acknowledge it in DevTools.
Here's a stub representing a longString: https://github.com/devtools-html/debugger.html/blob/30b0065dceb3933e116df1a15635c4bacf79f4b2/packages/devtools-reps/src/reps/stubs/long-string.js#L20-L27

I guess we could re-use it here for `"data-test"`

Here's the result we get for `var div = document.createElement("div"); div.setAttribute("data-test","a ".repeat(5000)); div;` (which creates a long string attribute)

```js
{
  "type": "object",
  "actor": "server1.conn0.child1/obj26",
  "class": "HTMLDivElement",
  "extensible": true,
  "frozen": false,
  "sealed": false,
  "ownPropertyLength": 0,
  "preview": {
    "kind": "DOMNode",
    "nodeType": 1,
    "nodeName": "div",
    "isConnected": false,
    "attributes": {
      "data-test": {
        "type": "longString",
        "initial": "a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a ",
        "length": 10000,
        "actor": "server1.conn0.child1/longString27"
      }
    },
    "attributesLength": 1
  }
}
```

Reply:
I think this is a naming issue; the usage of this stub is simply to have an attribute > max (50); I used the `NodeWithLongStringAttribute` to test a true LongString